tests/sqllogictests/suites/query/case_sensitivity/ddl.test

```diff
@@ -1,3 +1,4 @@
+#need onlyif mysql tag, because http test can not keep a session
```

Comment:
http test can keep session settings, each test file is a session.

Reply:
It feels like there are some other unknowns here, and I didn't add the `onlyif mysql` at the beginning. 

But I found that it affected the presentation of other test results.

Let't add it again, and observe the ci result.